src/templates/static/trading_platform/mt5_types_of_accounts.html.tt

```diff
@@ -140,7 +140,7 @@
                 <p>[% l('Is the ability to control a large amount of money using a minimal capital from yourself. Where does the rest of the money come from? Your trusted broker provides the rest.') %]</p>
                 <p>[% l('Let’s say you fork out a capital of USD 50 and we’re offering a leverage of 1:1,000. How much money, or margin, is available to you when you start trading?') %]</p>
                 <div class="box bordered">
-                    <p class="secondary-color">USD 50 x 1,000 = <strong>USD 50,000 [% l('(is the margin available)') %]</strong></p>
+                    <p class="secondary-color text-normal">USD 50 x 1,000 = <strong>USD 50,000 [% l('(is the margin available)') %]</strong></p>
```

Comment:
maybe remove the <strong> tag instead of declaring new class `text-normal` ?

Reply:
Noted, removed class and tag as suggested in commit 40ca683.